include/commonpp/core/LoggingInterface.hpp

```diff
@@ -61,9 +63,34 @@ UNUSED_ATTR static LoggingLevel to_severity(int lvl) noexcept
     return info;
 }
 
+static inline size_t to_syslog_level(commonpp::LoggingLevel level) noexcept
+{
+    switch (level)
+    {
+    case trace:
+        return 7;
+    case debug:
+        return 7;
+    case info:
+        return 6;
+    case warning:
+        return 4;
+    case error:
+        return 3;
+    case fatal:
+        return 2;
+    default:
```

Comment:
hum, I would default to info no ?

Reply:
It's kind of a moot point really. If we add a new log level (unlikely I think) I'd like to know something changed, and seeing fatal instead of info might alert people to the fact.

Reply:
fair enough